src/main.rs

```diff
@@ -174,12 +174,28 @@ fn select_emoji(colors: Colors) -> Result<Option<String>, Box<dyn Error>> {
 }
 
 fn install_hook(color_scheme: ColorScheme) -> Result<(), Box<dyn Error>> {
-    let mut file = File::create(HOOK_PATH)?;
     let color_scheme_arg = match color_scheme {
         ColorScheme::Light => "--color-scheme light",
         ColorScheme::Dark => "--color-scheme dark",
     };
-    let content = HOOK_CONTENT_TEMPL.replace("{color_scheme_arg}", color_scheme_arg);
+    let content = HOOK_CMD_TEMPL.replace("{color_scheme_arg}", color_scheme_arg);
+    let mut file = match OpenOptions::new()
+        .write(true)
+        .create_new(true)
+        .open(HOOK_PATH)
+    {
+        Ok(f) => f,
+        Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {
+            eprintln!(
+                "Failed to create `{HOOK_PATH}` as it already exists. \
+                Please either or remove it and re-run `gimoji -i` or \
```

Comment:
the first "or" doesn't belong there. and the 2nd or could use a comma

Reply:
didn't realize you'll actually review the PR so I had it setup for automatic merge. I'll fix it now..

Reply:
https://github.com/zeenix/gimoji/pull/102